uaclient/config.py

```diff
@@ -115,19 +115,19 @@ def entitlements(self):
 
         self._entitlements = {}
         contractInfo = machine_token["machineTokenInfo"]["contractInfo"]
-
+        tokens_by_name = dict(
+            (e["Type"], e["Token"])
```

Comment:
So this part of the response has the `Type` key capitalised, but the one we use in the next logical line has it lower-case? Is consistency something we could ask of ua-contracts, or has the ship sailed on changing the shape of the API response?

Reply:
I think it's worth a bug against ua-contracts on that, since we are the only consumer of that data. We can make this branch work with the lower() value in either case.

Reply:
Filed https://github.com/CanonicalLtd/ua-contracts/issues/645